src/main/java/tasks/Deadline.java

```diff
@@ -0,0 +1,31 @@
+package tasks;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task{
+
+    private LocalDate time;
```

Comment:
```suggestion
    private LocalDate date;
```

Reply:
Since I'm using dates, naming it date does make more sense. Thanks!